evmos/assetlist.json

```diff
@@ -100,6 +100,33 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/evmos/images/neok.svg"
         }
       ]
+    },
+    {
+      "description": "The token of Teledisko DAO.",
+      "denom_units": [
```

Comment:
Possible to add type.asset and address?

Reply:
I looked at the examples and couldn't find an explanation about these fields. Is it correct to assume that `type.asset` should be `erc20` and and `address` just `0x1cFc8f1FE8D5668BAFF2724547EcDbd6f013a280` ?

Reply:
Yes, I would assume so